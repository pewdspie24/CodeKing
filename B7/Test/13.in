68
7 4067759141
973492542 741686784
756483625 214915603
665401016 852987027
274055059 530714166
314629133 251776879
808949528 133001631
153678680 28575466
2 3896544923
546113316 552578675
514448545 439792829
8 4290056824
615668627 422468586
281787795 917773884
88365599 755294362
455264658 255381095
90251180 16403257
9646720 79269966
72809490 935721858
723085664 422129184
6 316253525
636306312 847446055
630418649 238777279
293379623 980742259
847866601 87981512
426577585 717857462
392819589 866711640
6 2011308633
86266343 570197957
551594236 658826922
75224930 897995210
339697850 312823580
35340554 607609414
915929950 556738264
9 838413408
618106336 96456231
944174229 569940327
534196452 564783874
728956390 953563927
181082842 47869661
235456330 228129475
115770419 280725920
762780281 897727796
294514881 425545546
9 2485664623
300681233 758186727
602220177 141951069
867547334 643921834
799583614 828528759
561373734 978817313
60971233 521805621
185937018 52243305
825292736 740136401
645990216 287498184
5 943453972
890846240 539605532
456837698 535361122
988511205 588142575
948019136 180312515
830032412 566523143
7 3682999179
914567653 795338016
298082347 187048492
360895046 113945426
427246433 964016196
129342156 704739130
588115116 230731623
498051269 739596779
10 500025500
133871126 443540614
865861930 689088510
330734504 737779980
134114644 383390262
739545918 329099857
155859552 139646121
510097815 551415588
857224205 236486256
855617702 720133279
693925859 51499489
7 2925028261
933441550 508500618
579040059 1885585
335565885 789776640
80594374 427603845
117518967 307719118
64546896 365161660
82802089 566571641
1 1991357313
999334974 285702378
9 4191794026
808566402 507511044
921082024 591621328
777871736 522891237
14275402 194090805
207611870 304894108
987354469 520689622
334368392 792480074
587746918 57759113
388405902 367025270
6 4200599651
735642707 390703059
700611822 16334715
774605562 206795817
35846163 847873818
1529519 658241356
901216409 425728042
4 49658740
752893154 4649475
650027286 725030577
713049499 834111165
807889963 56168682
1 1926177784
32531751 41528615
1 4228912019
856501735 917730680
1 2309623340
979755792 71013880
8 3613755185
473963968 997756191
374797578 161849628
290736607 169883813
796117954 788161482
670697556 844681453
498614993 517153569
417175623 291066710
644139772 795191967
5 2455911231
760069190 354055137
798762280 125766643
174429052 911384613
72299927 696595725
485866585 405318582
5 82357255
779641850 217134006
611949308 98596433
451387706 2455225
239861213 265702374
31710716 491069024
1 3100848414
144905069 76771638
9 2984550731
237307224 47925516
64834121 953465566
499791214 636721330
627628923 78470221
812240851 168146598
369340805 852292588
399798114 432542987
866029846 964712837
487641076 586865145
8 35350909
468235208 311576607
651977899 483055122
239585543 811265142
952103844 16073633
500531109 657417150
434429754 135323617
43302920 777785329
182561412 357966284
10 2269862658
248875325 733492978
98700019 498858559
5228310 103193558
480273393 592920161
610256467 487687583
65177056 96322544
141353926 276356772
172443147 382562228
905441353 737718196
250886790 223121013
2 3107112886
721667272 752318042
950396948 374539120
6 3403917853
733756499 187176112
678389016 141236406
377956033 292879083
614786464 116571295
454341823 119623966
342892685 260237717
1 964586499
436947295 155197768
2 1741806863
379930923 483516994
478246133 574369258
4 501290423
51231938 653140929
171931568 372107349
778418086 571450795
212277256 904994015
6 1176769656
321097152 288888140
8768693 892745617
143344693 998052292
526515859 38318995
317198302 155196817
422002576 291863500
2 61759527
847025944 520299540
224025323 640156311
7 1763076621
526541039 370886
215616697 137618079
594753068 111540006
873243624 367940965
832975450 543190695
90500444 978874479
398030534 680019235
10 2515827355
783904104 315731788
371492559 72071418
115840229 20101424
32720588 199196116
793898178 553001777
389714450 901730183
190296342 173383337
1304196 812142144
558228625 498336890
377429293 927138703
10 215806286
361957508 644574424
855188354 70913501
58896697 431703526
488752954 905393627
287188363 420002343
814796282 976951025
473119850 138787885
177450713 544378915
344942334 288156414
988022961 100684425
10 3088664823
345805678 451459766
550680745 115149335
307536457 760518513
336529392 180457478
920382639 892117097
979117325 271675030
733371580 744285238
852587599 399245514
226664926 400708970
831425073 629912498
5 3815224251
677347849 162985608
56913304 591548854
20919776 470997677
733766278 311987507
392934020 680203979
9 3044314616
207547529 594051753
905406281 86623976
279032027 717785824
329851701 267166096
512688428 476503137
67434181 433841137
185241614 931949217
511975172 899398751
748214863 558529728
4 3675136417
270690427 148618213
616688292 642599065
770005308 507851934
913805340 46195031
3 2402237366
931210500 292521939
574449122 693084237
750941759 819729311
10 2244564125
869209912 750172
967040038 191507022
729860133 61636930
383962307 992861490
894396177 219681084
218380074 348275498
115799810 84895570
818367264 271959400
395241420 198059483
672347766 111350226
4 3478515600
239325865 492491956
767130204 935421099
62746779 291452974
412890519 773190828
5 2012964051
607481682 237116074
398546619 681593458
568098262 653964093
792519293 767022090
239222169 123807260
8 432101150
894356694 26469084
146915424 273954636
165660490 531092884
590883725 975103796
440269213 522921297
401246516 825679207
134264894 969965888
905688948 638568468
1 1165083977
73901968 177429003
4 3305188603
100399722 854657947
474904281 690050589
775281601 546319999
551281632 996555241
4 2502333754
229792907 445896293
798337743 805858989
668620654 908602586
848179516 569519223
9 1640712952
384833336 235470252
944804079 776392518
546140187 908070223
137720646 653741526
720015379 975421646
290902588 804077473
402437519 712254988
747245043 290964251
369045394 92704356
10 2192016778
739530330 346850169
695695964 781373819
826238180 366752534
99428910 352578424
809338846 891622137
788733544 503777446
607204092 8994067
882559500 437549089
466214040 833401456
145574896 904260358
9 1522538772
184867882 112794860
656073450 166381074
809837907 888998673
560779818 947045415
26696260 641464699
655616069 15405827
699616708 390542594
646484663 558396143
284096449 257109335
1 2513343491
660547531 340032088
5 872973115
5755829 478188763
781277008 452738333
738587290 721917997
830135040 913682528
391621828 397797464
2 3464561968
566736772 583898523
608599583 525499158
4 2727162683
857684938 645367766
966130154 257472947
555881411 621876605
645469244 889729439
10 2389331082
254095334 967892546
696962813 691601567
910654 368834921
321534322 523973516
793073669 271404579
782451574 51463565
123720728 469955279
445659277 337722323
50335623 164706697
592464036 475450970
5 3729158974
91431809 556554671
604271631 958152682
340065661 312498336
74674443 509980779
144079750 381454918
3 3619819116
533144278 122694788
671257364 434862235
638745803 868625856
3 1324943790
708532746 215313249
43225492 357922281
492785015 438851487
5 2852732987
124415084 859166090
578707988 33572113
48158281 388390626
475609192 138116076
216842976 893143402
10 4213997295
653585066 431145643
230820283 862772304
762634582 75519464
399538233 387553525
791801311 534068211
196118268 610500234
52991080 587634089
729244404 750956944
388549297 216371814
319118793 686609661
5 772709256
149070243 324783204
727524307 611281711
256904756 273247445
667691607 505203826
904271002 865538189
10 3267576138
801090059 596668393
668414007 955511060
579703514 843299057
551101089 704675670
120316433 103764729
984901844 757559869
273100160 196536365
121016562 736238100
231112395 962202065
338982470 904184421
5 3082079864
573592470 239790460
668141091 914591589
304676049 523218796
849629589 204330504
106875857 554920907
5 2109241450
616306700 258872019
745771333 698921410
954552289 940956015
609057432 808975594
413156919 494701407
9 1739304040
645586880 368974004
568080230 29651186
304207963 255660133
670681178 728875701
11792989 625171506
245034051 397488476
139705012 164874142
891297750 220094658
874539673 674840730
9 3791220755
305596360 82500753
51677475 966077453
365088976 359381713
766346457 896542612
448026469 246856995
220888207 888514078
654167298 110665411
685736220 218104165
255297890 703214431
2 2502013053
4435212 574718368
884828377 567381617
9 2118797519
776316572 106779084
357117112 318379233
581289342 838051123
493323071 244540543
779940158 131712255
125808567 279155282
41739815 167776026
195202013 236707590
788806257 966088690
