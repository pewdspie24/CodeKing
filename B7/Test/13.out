6
46
13
1
5
1
5
1
9
1
5
3
8
6
1
MISSION FAILED!
MISSION FAILED!
3
15
4
1
45
5
1
10
5
6
3
MISSION FAILED!
1
3
1
3
5
1
4
7
4
5
4
3
MISSION FAILED!
5
1
MISSION FAILED!
13
7
3
4
3
7
2
36
4
4
115
9
3
6
6
1
12
5
5
3
7
7
3
